src/share/util/shr_scam_mod.F90

```diff
@@ -271,6 +271,10 @@ subroutine shr_scam_getCloseLatLonNC(ncid, targetLat,  targetLon, closeLat, clos
    if ( allocated(londimnames) ) deallocate(londimnames)
    if ( allocated(vars)        ) deallocate( vars )
 
+   if (nlat .eq. nlon) then !-- Enforce for SE grids
```

Comment:
This doesn't seem like a reliable test of what grid you are on - can you do better?